dymos/examples/finite_burn_orbit_raise/test/test_two_burn_orbit_raise_linkages.py

```diff
@@ -247,6 +247,523 @@ def test_two_burn_orbit_raise_gl_rk_gl_constrained(self):
+
+        p.set_val('traj.burn1.t_initial', value=0.0)
+        p.set_val('traj.burn1.t_duration', value=2.25)
+
+        p.set_val('traj.burn1.states:r',
+                  value=burn1.interpolate(ys=[1, 1.5], nodes='state_input'))
+        p.set_val('traj.burn1.states:theta',
+                  value=burn1.interpolate(ys=[0, 1.7], nodes='state_input'))
+        p.set_val('traj.burn1.states:vr',
+                  value=burn1.interpolate(ys=[0, 0], nodes='state_input'))
+        p.set_val('traj.burn1.states:vt',
+                  value=burn1.interpolate(ys=[1, 1], nodes='state_input'))
+        p.set_val('traj.burn1.states:accel',
+                  value=burn1.interpolate(ys=[0.1, 0], nodes='state_input'))
+        p.set_val('traj.burn1.states:deltav',
+                  value=burn1.interpolate(ys=[0, 0.1], nodes='state_input'), )
+        p.set_val('traj.burn1.controls:u1',
+                  value=burn1.interpolate(ys=[-3.5, 13.0], nodes='control_input'))
+
+        p.set_val('traj.coast.t_initial', value=2.25)
+        p.set_val('traj.coast.t_duration', value=3.0)
+
+        p.set_val('traj.coast.states:r',
+                  value=coast.interpolate(ys=[1.3, 1.5], nodes='state_input'))
+        p.set_val('traj.coast.states:theta',
+                  value=coast.interpolate(ys=[2.1767, 1.7], nodes='state_input'))
+        p.set_val('traj.coast.states:vr',
+                  value=coast.interpolate(ys=[0.3285, 0], nodes='state_input'))
+        p.set_val('traj.coast.states:vt',
+                  value=coast.interpolate(ys=[0.97, 1], nodes='state_input'))
+        p.set_val('traj.coast.states:accel',
+                  value=coast.interpolate(ys=[0, 0], nodes='state_input'))
+
+        p.set_val('traj.burn2.t_initial', value=5.25)
+        p.set_val('traj.burn2.t_duration', value=1.75)
+
+        p.set_val('traj.burn2.states:r',
+                  value=burn2.interpolate(ys=[1.8, 3], nodes='state_input'))
+        p.set_val('traj.burn2.states:theta',
+                  value=burn2.interpolate(ys=[3.2, 4.0], nodes='state_input'))
+        p.set_val('traj.burn2.states:vr',
+                  value=burn2.interpolate(ys=[.5, 0], nodes='state_input'))
+        p.set_val('traj.burn2.states:vt',
+                  value=burn2.interpolate(ys=[1, np.sqrt(1 / 3)], nodes='state_input'))
+        p.set_val('traj.burn2.states:accel',
+                  value=burn2.interpolate(ys=[0.1, 0], nodes='state_input'))
+        p.set_val('traj.burn2.states:deltav',
+                  value=burn2.interpolate(ys=[0.1, 0.2], nodes='state_input'))
+        p.set_val('traj.burn2.controls:u1',
+                  value=burn2.interpolate(ys=[1, 1], nodes='control_input'))
+
+        p.run_driver()
+
+        assert_near_equal(p.get_val('traj.burn2.timeseries.states:deltav')[-1],
+                          0.3995,
+                          tolerance=2.0E-3)
+
+        # test ODE output wildcard matching in solve_ivp
+        traj.simulate()
```

Comment:
I think we need to capture this as `exp_out` and then check to make sure the expected outputs are in the timeseries, just as a test of the SolveIVP case.

Reply:
I can add back the plotting from the original test.

Reply:
I added a comparison between the exp_out timeseries and the solution time series

Reply:
Hypothetically, if those outputs were added to the timeseries output but weren't picking up values correctly, they'd both be the same but incorrect.  Since we know pos_x = r * cos(theta) and pox_y = r * sin(theta), and we know that state and control timseries outputs are working based on other tests, it's probably best to test against those equations.